front/src/common/HelpModal/HelpModal.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
+import ModalHeaderSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF';
+
+const expectedFileFirstLine = '<!-- OSRD Help Markdown File -->';
+const defaultText = '# contact@osrd.fr';
+
+function HelpModal() {
+  const { t } = useTranslation('home/navbar');
+  const [helpModalText, setHelpModalText] = useState<string>('');
+
+  useEffect(() => {
+    const checkHelpMarkdown = async () => {
+
+      try {
+        const response = await fetch('/help.md');
+
+        if (!response.ok) {
+          setHelpModalText(defaultText);
+          return;
+        }
+
+        const text = await response.text();
+        setHelpModalText(
+          text.startsWith(expectedFileFirstLine)
+            ? text.replace(expectedFileFirstLine, '')
+            : defaultText
+        );
+      } catch {
+        setHelpModalText(defaultText);
+      }
+    };
+
+    checkHelpMarkdown();
+  }, []);
+
+  return (
+    <div className="informations-modal h-100">
```

Comment:
Not sure the `height: 100%` here will work as there is a`max-height: 75vh` in the `informations-modal` class